pyproject.toml

```diff
@@ -49,7 +49,7 @@ test = [
     "psutil",
     "pytest>=4.6.0",
     "pytest-doctestplus",
-    "crds>=11.16.14",
+    "crds>=11.17.1",
```

Comment:
why the bump in CRDS version?

Reply:
Thanks for giving this a look.

Earlier version of crds use `DataModel`. See:
https://github.com/spacetelescope/crds/releases/tag/11.17.1
and:
https://github.com/spacetelescope/crds/pull/938